pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
Should not be downgraded. Are you running the latest pnpm version?

Reply:
probably not.